sections/futures/Trade/Trade.tsx

```diff
@@ -255,7 +255,10 @@ const Trade: React.FC<TradeProps> = () => {
 					{t('futures.market.trade.button.deposit')}
 				</MarketActionButton>
 				<MarketActionButton
-					disabled={futuresMarketsPosition?.remainingMargin?.lte(zeroBN)}
+					disabled={
+						!futuresMarketsPosition?.remainingMargin ||
```

Comment:
The code will be easier to understand if we use `isWalletConnected` to check the status of the wallet connection. https://github.com/Kwenta/kwenta/blob/v2-dev/store/wallet/index.ts#L35-L37